restler-core/src/main/java/net/researchgate/restdsl/dao/MongoServiceDao.java

```diff
@@ -42,7 +42,10 @@
 public class MongoServiceDao<V, K> implements PersistentServiceDao<V, K> {
     private static final Logger LOGGER = LoggerFactory.getLogger(MongoServiceDao.class);
 
-    private static final String QUERY_KEY = "queries.shapes.%s.%%H";
+    private static final String METRICS_SEPARATOR = "__";
```

Comment:
This is very specific to our implementation of how we encode metrics but other users will use it the graphite way normally